### PR TITLE
Colors assets

### PR DESCRIPTION
**Pull request by piensaenpixel**
@xavijam i dont know if is the best solution but could you take a look ?

### CODE REVIEW

File: src/scss/cdb-utilities/helpers.scss [hunk 1]
```diff
@@ -36,6 +37,17 @@
 .u-bSpace--m {
   margin-bottom: 8px;
 }
+.u-textSecondary {
```
Comment (xavijam) [inline src/scss/cdb-utilities/helpers.scss]: Should we add `.u-textPrimary`?

File: src/scss/cdb-utilities/helpers.scss [hunk 2]
```diff
@@ -36,6 +37,17 @@
...
+  color: $cTypo2;
+}
+.u-textSubtitle {
```
Comment (xavijam) [inline src/scss/cdb-utilities/helpers.scss]: In my opinion we shouldn't generate those classes with element types. We could follow name variables, what about...?:

``` css
.u-textPrimary {}
.u-textSecondary {}
.u-textTertiary {}
.u-textQuaternary {}
```

cc @javierarce

File: src/scss/cdb-utilities/helpers.scss (hunk 1 above)
Comment (piensaenpixel) [inline src/scss/cdb-utilities/helpers.scss]: For me textPrimary is default text

File: src/scss/cdb-utilities/helpers.scss (hunk 2 above)
Comment (piensaenpixel) [inline src/scss/cdb-utilities/helpers.scss]: okis go ahead

Comment (javierarce) [inline src/scss/cdb-utilities/helpers.scss]: As long as we don't go beyond four options, I'm ok (in that case we should rethink this kind of naming)

Comment (xavijam) [inline src/scss/cdb-utilities/helpers.scss]:
``` css
.u-mainTextColor {}
.u-secondaryTextColor {}
.u-aux1TextColor {}
.u-aux2TextColor {}
```

**Comment by piensaenpixel** [comment]
@xavijam @javierarce could you take a look?

**Comment by piensaenpixel** [comment]
final version? @xavijam @javierarce

**Comment by javierarce** [comment]
I have a concern with the use of the word `Typo`, which means `a typographical error.`

File: src/scss/cdb-utilities/helpers.scss [hunk 3]
```diff
@@ -37,6 +38,25 @@
...
 }
 
+// Colors
+// Main Text Color is default color 
+// ----------------------------------------------
+
+.u-mainTextColor {
+  color: $cMainDark;
+}
+.u-textSecondary {
```
Comment (xavijam) [inline src/scss/cdb-utilities/helpers.scss]: .u-secondaryTextColor

**Comment by piensaenpixel** [comment]
@xavijam @javierarce

File: src/scss/cdb-utilities/helpers.scss [hunk 4]
```diff
@@ -37,6 +38,25 @@
...
 
+// Colors
+// Main Text Color is default color 
+// ----------------------------------------------
+
+.u-mainTextColor {
+  color: $cMainText;
+}
+.u-secondaryTextColor {
+  color: $cTypo2;
```
Comment (xavijam) [inline src/scss/cdb-utilities/helpers.scss]: This color doesn't exist.

File: src/scss/cdb-utilities/helpers.scss [hunk 5]
```diff
@@ -37,6 +38,25 @@
...
+}
+.u-altTextColor {
+  color: $cTypo3;
```
Comment (xavijam) [inline src/scss/cdb-utilities/helpers.scss]: This color doesn't exist.

File: src/scss/cdb-utilities/helpers.scss [hunk 6]
```diff
@@ -37,6 +38,25 @@
...
+}
+.u-hintTextColor {
+  color: $cTypo4;
```
Comment (xavijam) [inline src/scss/cdb-utilities/helpers.scss]: This color doesn't exist.

File: src/scss/cdb-utilities/defaults.css.scss [hunk 7]
```diff
@@ -3,6 +3,10 @@
 
 @import '../cdb-variables/colors';
 
+body {
+  color: $cTypoMain;
```
Comment (xavijam) [inline src/scss/cdb-utilities/defaults.css.scss]: This variable doesn't exist.

File: src/scss/cdb-components/menu.scss [hunk 8]
```diff
@@ -119,13 +119,12 @@
 .CDB-NavSubmenu-item.is-active {
   .CDB-NavSubmenu-Link {
     border-bottom: 1px solid $cTypo3;
```
Comment (xavijam) [inline src/scss/cdb-components/menu.scss]: A border should not have a typography color.

File: src/scss/cdb-components/loader.scss [hunk 9]
```diff
@@ -105,12 +105,12 @@ This is the generic loader for widgets, maps, components, ...
     background: $cWhite;
   }
   &.is-grey {
-    background: linear-gradient(to right, $cMainDark 10%, rgba($cWhite, 0) 42%);
+    background: linear-gradient(to right, $cMainBg 10%, rgba($cWhite, 0) 42%);
     &::before {
       background: $cTypo2;
```
Comment (xavijam) [inline src/scss/cdb-components/loader.scss]: a background should not have a typography color.

Comment (piensaenpixel) [inline src/scss/cdb-components/loader.scss]: MainBg is not a typography color

File: src/scss/cdb-components/colors.scss [hunk 10]
```diff
@@ -89,13 +89,13 @@
 
 
 .Color-Type02 {
```
Comment (xavijam) [inline src/scss/cdb-components/colors.scss:91]: Do we need this?

**Comment by xavijam** [comment]
It looks :+1:, :clap:. I've only commented one thing. But ready to go!

**Pull request merged**